src/actor/component.rs

```diff
@@ -131,14 +143,25 @@ impl Component {
         &self,
         operation: impl AsRef<str>,
         payload: Option<impl AsRef<[u8]>>,
+    ) -> anyhow::Result<Result<Option<Vec<u8>>, String>> {
+        self.call_with_context(operation, payload, vec![]).await
+    }
+
+    /// Instantiate a [Component] producing an [Instance] and invoke an operation on it using [Instance::call]
+    #[instrument(skip(operation, payload, call_context))]
+    pub async fn call_with_context(
+        &self,
+        operation: impl AsRef<str>,
+        payload: Option<impl AsRef<[u8]>>,
+        call_context: impl AsRef<[u8]>,
```

Comment:
```suggestion
        call_context: C
```

Reply:
See above about replacing the generic in a future PR